Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -907,20 +910,23 @@ public static void CreateUpdateRunSetFromJSON(RunsetExecutor runsetExecutor, Run
                         {
                             mailOperation.Comments = runsetOperationConfigMail.Comments;
                         }
-                        if (runsetOperationConfigMail.IncludeAttachmentReport != null && runsetOperationConfigMail.IncludeAttachmentReport == true)
+                        if (runsetOperationConfigMail.IncludeAttachmentReport != null)
                         {
-                            if (mailOperation.EmailAttachments.Count == 0)
+                            if (runsetOperationConfigMail.IncludeAttachmentReport == true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4955374)